Require support tickets to be *initiated* by community reps (#797)
2i2c-org/team-compass: projects/managed-hubs/support.md

When a new ticket lands in Freshdesk under the support group and it is not an incident, we aim to respond within 24 working hours with a suggested next action. The next steps should be followed when resolving a ticket:

> 1. `Who:Support steward`

Reviewer: ```suggestion
1. `Who: Support steward`
```

>    First, we determine if the person *initiating* the support ticket is *authorized* to do actually do so. While we may interact with many folks

Reviewer 2: ```suggestion
   First, we determine if the person *initiating* the support ticket is *authorized* to actually do so. While we may interact with many folks
```

   from a community during resolution of a ticket, we constrain who can *initiate* a ticket to {term}`Community Representative`s only. This prevents our
   support staff from being overwhelmed by tickets that need to be handled elsewhere. If the person *initiating* the ticket is not a community
   representative, the support steward should cc the community representatives, and ask for approval. The support steward *may* choose to use the following

>    email template:

Reviewer 2: ```suggestion
   email template also available as a [Canned Response](https://2i2c.freshdesk.com/a/admin/canned_responses/folders):
```

I added this into our canned responses on Freshdesk as well so it's really easy to find.

   > Hello <names of community representatives>,
   >
   > I'm cc'ing you on this support ticket we received from a member of your community. To streamline our support process, 2i2c is accepting

>    > support requests only from communtity representatives. Can you read through the request, and let us know how you wish to proceed?

Reviewer 2: ```suggestion
   > support requests only from community representatives. Can you read through the request, and let us know how you wish to proceed?
```

   If the support stewards (both of them) do not have the capacity to resolve the `critical` issue (ie. working on another `critical` issue, being out of their working time, etc.), they should ping the **Engineering Manager** (or the delegated person) so they can secure resources to resolve that issue on the fly (see step 7 below).

   The support steward **should not** work on issues with impact lower than `critical` (unless they are assigned as part of the "planned" reactive work in the context of a running sprint (see step 6 below).

5. `Who: Partnerships representative and the Engineering Manager (or respective delegates)`

   **Revisit the impact metadata**. Once a week (at minimum) the [support view in the **Eng & Prod** board](https://github.com/orgs/2i2c-org/projects/22/views/47) should be revisited to validate the impact level on support-related issues. Currently, we allocate a 30-minute working session every Wednesday (open to everyone to participate) to perform such impact revision and further prioritization ("planned" reactive) every other week (see step 7 for more details).

6. `Who: Support steward`

>    **Add a reference/link to the created engineering issue inside the Freshdesk ticket**. You can use an internal note or make it public when you communicate back to the Community Representative in step 6. Also, move the status of the ticket to the "Pending" state.

Reviewer: ```suggestion
   **Add a reference/link to the created engineering issue inside the Freshdesk ticket and set ticket status to Pending**. You can use a private note or make it public when you communicate back to the Community Representative in the next step.
```

Author: although this made me realize that I'm going to have to touch this anyway, as the step names have changed. So I'll just incorporate your suggestion into it too @consideRatio when i edit it!
